File: platform-user-service/src/main/java/com/talentprogram/LdPlatformUserService/dto/UserDTO.java
package com.talentprogram.LdPlatformUserService.dto;
import java.util.Set;

public record UserDTO(
  Long id,
  String username,
  String fullName,
  String title,
  Long managerId,
  Set<String> roles /*roles review TODO*/
)
{
    
}

File: platform-webapp/src/app/model/UserDto.ts
import { Role } from "./Role";

export interface UserDto {
    id: number,
    username: string,
    fullName: string,
    title: string,
    managerId: number,
    roles: Set<Role> //Set<"EMPLOYEE" | "GUEST" | "ADMIN" | "MANAGER">//
}